### PR TITLE
refactor: Consolidate file reading and enhance status reporting in checks

### PR DESCRIPTION
**Pull request by dz0ny**
Adds tests for the firewall and autologin; creates a test harness for system operations and makes failures more detailed in CLI.

### CODE REVIEW

File: checks/autologin.go [hunk 1]
```diff
@@ -1,10 +1,11 @@
 package checks
 
 import (
-	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
+
+	"paretosecurity.com/auditor/shared"
```
Comment (zupo) [inline checks/autologin.go:8]: Can we stop using the "auditor" term? We no longer have the "updated" so having "auditor" does not make sense.

Comment (dz0ny) [reply checks/autologin.go:8]: checker? It needs to be something.

Comment (zupo) [reply checks/autologin.go:8]: linux?

Comment (dz0ny) [reply checks/autologin.go:8]: Actually we now have public repo so module name is repo name.

**Pull request merged**